src/gc/gc.cpp

```diff
@@ -36090,6 +36090,11 @@ void GCHeap::GetMemoryInfo(uint32_t* highMemLoadThreshold,
     *lastRecordedFragmentation = gc_heap::last_gc_fragmentation;
 }
 
+int64_t GCHeap::GetTotalAllocatedBytes(bool precise)
```

Comment:
I suspect that the signature of this method may be different once you actually implement this.